roles/openshift_openstack/library/os_service_catalog.py

```diff
@@ -0,0 +1,177 @@
+        id: 4cae4dcabe0a4914a6ec6dabd62490ba
+        internalURL: http://172.16.0.9:8777
+        publicURL: https://mycloud.org:13777
+        region: regionOne
+      metric:
+      - adminURL: http://172.16.0.9:8041
+        id: 29bcecf9a06f40f782f19dd7492af352
+        internalURL: http://172.16.0.9:8041
+        publicURL: https://mycloud.org:13041
+        region: regionOne
+      network:
+      - adminURL: http://172.16.0.9:9696
+        id: 5d5785c9b8174c21bfb19dc3b16c87fa
+        internalURL: http://172.16.0.9:9696
+        publicURL: https://mycloud.org:13696
+        region: regionOne
+      object-store:
+      - adminURL: http://172.17.0.9:8080
+        id: 031f1e342fdf4f25b6099d1f3b0847e3
+        internalURL: http://172.17.0.9:8080/v1/AUTH_6d2847d6a6414308a67644eefc7b98c7
+        publicURL: https://mycloud.org:13808/v1/AUTH_6d2847d6a6414308a67644eefc7b98c7
+        region: regionOne
+      orchestration:
+      - adminURL: http://172.16.0.9:8004/v1/6d2847d6a6414308a67644eefc7b98c7
+        id: 1e6cecbd15b3413d9411052c52b9d433
+        internalURL: http://172.16.0.9:8004/v1/6d2847d6a6414308a67644eefc7b98c7
+        publicURL: https://mycloud.org:13004/v1/6d2847d6a6414308a67644eefc7b98c7
+        region: regionOne
+      placement:
+      - adminURL: http://172.16.0.9:8778/placement
+        id: 1f2551e5450c4bd6a9f716f92e93a154
+        internalURL: http://172.16.0.9:8778/placement
+        publicURL: https://mycloud.org:13778/placement
+        region: regionOne
+      volume:
+      - adminURL: http://172.16.0.9:8776/v1/6d2847d6a6414308a67644eefc7b98c7
+        id: 38e369a0e17346fe8e37a20146e005ef
+        internalURL: http://172.16.0.9:8776/v1/6d2847d6a6414308a67644eefc7b98c7
+        publicURL: https://mycloud.org:13776/v1/6d2847d6a6414308a67644eefc7b98c7
+        region: regionOne
+      volumev2:
+      - adminURL: http://172.16.0.9:8776/v2/6d2847d6a6414308a67644eefc7b98c7
+        id: 113a0bff9f2347b6b8774407a1c8d572
+        internalURL: http://172.16.0.9:8776/v2/6d2847d6a6414308a67644eefc7b98c7
+        publicURL: https://mycloud.org:13776/v2/6d2847d6a6414308a67644eefc7b98c7
+        region: regionOne
+      volumev3:
+      - adminURL: http://172.16.0.9:8776/v3/6d2847d6a6414308a67644eefc7b98c7
+        id: 9982c0afd28941a19feb1ffb13b91daf
+        internalURL: http://172.16.0.9:8776/v3/6d2847d6a6414308a67644eefc7b98c7
+        publicURL: https://mycloud.org:13776/v3/6d2847d6a6414308a67644eefc7b98c7
+        region: regionOne
+'''
+
+
+def main():
+    ''' Main module function '''
+    module = AnsibleModule(argument_spec={}, supports_check_mode=True)
+
+    if not HAS_SHADE:
+        module.fail_json(msg='shade is required for this module')
+
+    try:
+        cloud = shade.openstack_cloud()
+    # pylint: disable=broad-except
+    except Exception:
+        module.fail_json(msg='Failed to connect to the cloud')
+
+    try:
+        sc = cloud.cloud_config.get_service_catalog()
```

Comment:
Tox/pylint complains about the `sc` name:

```
roles/openshift_openstack/library/os_service_catalog.py:159: [C0103(invalid-name), main] Invalid variable name "sc"
error: lint error 16.
ERROR: InvocationError for command '/home/travis/build/openshift/openshift-ansible/.tox/py27-pylint/bin/python setup.py lint' (exited with code 1)
```

Can we appease the poor bot by renaming it to `catalog` or some such?